Miniforge3/construct.yaml

```diff
@@ -16,8 +16,11 @@ write_condarc: True
 # and keep the same with Miniconda
 keep_pkgs: True
 license_file: ../LICENSE
-# Blocked until https://github.com/mamba-org/mamba/issues/1497 is fixed
-# transmute_file_type: .conda
```

Comment:
I don't think we need to transmute anymore since packages are being repackaged as conda natively.